### PR TITLE
Zero duration offering

### LINKED ISSUE #2840: An Offering Of Zero Hours and Zero Minutes Should Not Be Saved
II found that it is possible to save an offering with a date and no duration whatsoever. Example below ...

Screen Shot 2022-05-04 at 10 08 58 AM

Since there is no way this could ever shown up on a calendar, we should require a duration for an offering. Or at least we should talk about requiring a duration for an offering.

### PR DESCRIPTION
**Pull request by stopfstedt**
fixes #2840

### CODE REVIEW

**Comment by dartajax** [comment]
I understand and approve of this idea but am unable to test this as a common PR - no edit buttons are available.

**Review by dartajax — APPROVED** [review]
I can't test this one way or the other on common since the edit functionality in offeringsnis not available. Code and logic seems fine.

**Comment by stopfstedt** [comment]
> I understand and approve of this idea but am unable to test this as a common PR - no edit buttons are available.

looks like common is in read-only mode b/c frontend/backend are out of sync.

**Review by jrjohnson — APPROVED** [review]

**Review by dartajax — CHANGES_REQUESTED** [review]
this occurs when I went in to modify an existing offering of zero hours and fifty minutes ...

Screen Shot 2022-05-17 at 3 58 53 PM

The duration msg didn't pop up and it shot the box down vertically

File: tests/integration/components/offering-form-test.js [hunk 1]
```diff
@@ -36,13 +36,6 @@ module('Integration | Component | offering form', function (hooks) {
     assert.ok(component.location.hasError);
   });
 
-  test('room validation succeeds on blank value', async function (assert) {
```
Comment (jrjohnson) [inline tests/integration/components/offering-form-test.js:39]: Can't tell why this test got removed, superfluous now?

**Comment by stopfstedt** [comment]
> this occurs when I went in to modify an existing offering of zero hours and fifty minutes ...
> 
> Screen Shot 2022-05-17 at 3 58 53 PM
> 
> The duration msg didn't pop up and it shot the box down vertically

can't reproduce. 0 hrs and 0 minute always ends up like this for me: 

image

(as it should be). can you confirm this is still an issue. if so, please provide detailed instructions on how your ended in that place, thanks!

**Comment by dartajax** [comment]
Steps ...

* go to https://demo.iliosproject.org/courses/2016/sessions/68578
* go to edit offering
Screen Shot 2022-06-10 at 10 38 25 AM

* the offering has to have 0 hrs and some amount of minutes as its current duration
* change the minutes value to zero 
Screen Shot 2022-06-10 at 10 42 16 AM

File: tests/integration/components/offering-form-test.js (hunk 1 above)
Comment (stopfstedt) [reply tests/integration/components/offering-form-test.js]: removal by accident. i added it back in. :heavy_check_mark:

**Review by dartajax — APPROVED** [review]
looks good to me

**Pull request merged**